source/triggers/aws-eventbridge.txt

```diff
@@ -96,22 +102,30 @@ Procedure
 
             Set the ``account_id`` field to the :guilabel:`AWS Account ID`
             that you copied from EventBridge and set the ``region`` field to
-            an AWS Region.
+            an AWS Region. 
+            
+            Optionally, you can serialize event objects into 
+            :manual:`Extended JSON format </reference/mongodb-extended-json>`. 
+            To enable Extended JSON, set the ``extended_json_enabled`` 
+            field to ``true``. By default, ``extended_json_enabled``  
+            is ``false``.
```

Comment:
Can we give a bit more _"why"_ here? If I'm following this guide it feels a bit jarring to have to switch to the manual to read more about why I would care either way. I think we could explain that EJSON preserves type information but may require additional work to process.

See my [previous comment](https://github.com/mongodb/docs-app-services/pull/259#discussion_r1042668609) for an example but feel free to adapt.